www/source/stylesheets/_demo.scss

```diff
@@ -177,6 +279,10 @@ a.previous-step-link {
   max-width: 300px;
 }
 
+.demo-step--pkg-img {
```

Comment:
We should probably remove this -- unless you plan on using it in subsequent demos?